packages/wrangler/src/pages/dev.ts

```diff
@@ -930,3 +830,203 @@ async function spawnProxyProcess({
 
 	return port;
 }
+
+/**
+ * Reconciles top-level & local development settings, with `pages dev`command line args taking
+ * precedence over `wrangler.toml` configuration
```

Comment:
Might be worth noting that this does not deal with bindings.

Reply:
fixed